assets/aircraft/js32.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "British Aerospace BAe-3200 Jetstream Super 31",
+    "icao": "JS32",
+    "engines": {
+        "number": 2,
+        "type": "S"
```

Comment:
Can't find `S` in [aircraft-format.md](https://github.com/openscope/openscope/blob/develop/documentation/aircraft-format.md), does it mean turboshaft? Maybe this should be added to the documentation?

Reply:
Okay, then I guess it should be `T` for both of my added aircraft? I'd also guess that the JS31 (not mine) needs changing as well?

Reply:
I have changed to `T` for both aircraft added in this PR.